SwiftTweaks/TweakCollectionViewController.swift

```diff
@@ -145,44 +145,44 @@ extension TweakCollectionViewController: TweakTableCellDelegate {
 }
 
 extension TweakCollectionViewController: TweakColorEditViewControllerDelegate {
-	func tweakColorEditViewControllerDidPressDismissButton(tweakColorEditViewController: TweakColorEditViewController) {
+	func tweakColorEditViewControllerDidPressDismissButton(_ tweakColorEditViewController: TweakColorEditViewController) {
 		self.delegate.tweakCollectionViewControllerDidPressDismissButton(self)
 	}
 }
 
 extension TweakCollectionViewController: TweakGroupSectionHeaderDelegate {
-	private func tweakGroupSectionHeaderDidPressFloatingButton(sectionHeader: TweakGroupSectionHeader) {
+	fileprivate func tweakGroupSectionHeaderDidPressFloatingButton(_ sectionHeader: TweakGroupSectionHeader) {
 		guard let tweakGroup = sectionHeader.tweakGroup else { return }
 
 		delegate.tweakCollectionViewController(self, didTapFloatingTweakGroupButtonForTweakGroup: tweakGroup)
 	}
 }
 
 private protocol TweakGroupSectionHeaderDelegate: class {
-	func tweakGroupSectionHeaderDidPressFloatingButton(sectionHeader: TweakGroupSectionHeader)
+	func tweakGroupSectionHeaderDidPressFloatingButton(_ sectionHeader: TweakGroupSectionHeader)
 }
 
 /// Displays the name of a tweak group, and includes a (+) button to present the floating TweakGroup UI when tapped.
 private final class TweakGroupSectionHeader: UITableViewHeaderFooterView {
 	static let identifier = "TweakGroupSectionHeader"
 
-	private let floatingButton: UIButton = {
-		let button = UIButton(type: .Custom)
-		let buttonImage = UIImage(swiftTweaksImage: .FloatingPlusButton).imageWithRenderingMode(.AlwaysTemplate)
-		button.setImage(buttonImage.imageTintedWithColor(AppTheme.Colors.controlTinted), forState: .Normal)
-		button.setImage(buttonImage.imageTintedWithColor(AppTheme.Colors.controlTintedPressed), forState: .Highlighted)
+	fileprivate let floatingButton: UIButton = {
```

Comment:
Ditto, these are not used outside of this type, please change them all back to `private`.